aliyun/src/main/java/org/apache/iceberg/aliyun/oss/OSSInputStream.java

```diff
@@ -44,10 +48,20 @@
   private long next = 0;
   private boolean closed = false;
 
+  private final Counter<Long> readBytes;
+  private final Counter<Integer> readOperations;
+
   public OSSInputStream(OSS client, OSSURI uri) {
```

Comment:
Should we deprecate this, too?

Reply:
Yes, I agree. Deprecating this would maintain the parity with the other InputStream classes (which have a package private constructor).